ui/raidboss/data/05-shb/trial/diamond_weapon-ex.js

```diff
@@ -0,0 +1,268 @@
+      netRegexDe: NetRegexes.startsUsing({ source: 'Diamant-Waffe', id: '5F9A', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Diamant', id: '5F9A', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ダイヤウェポン', id: '5F9A', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'East claw -> AoE',
+        },
+      },
+    },
+    {
+      id: 'DiamondEx Adamant Purge West Diamond Flash',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FA5', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Diamant-Waffe', id: '5FA5', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Diamant', id: '5FA5', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ダイヤウェポン', id: '5FA5', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'West claw -> Stack',
+        },
+      },
+    },
+    {
+      id: 'DiamondEx Adamant Purge East Diamond Flash',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FA4', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Diamant-Waffe', id: '5FA4', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Diamant', id: '5FA4', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ダイヤウェポン', id: '5FA4', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'East claw -> Stack',
+        },
+      },
+    },
+    {
+      id: 'DiamondEx Adamant Purge West Homing Laser',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FA3', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Diamant-Waffe', id: '5FA3', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Diamant', id: '5FA3', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ダイヤウェポン', id: '5FA3', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'West claw -> Spread',
+        },
+      },
+    },
+    {
+      id: 'DiamondEx Adamant Purge East Homing Laser',
+      netRegex: NetRegexes.startsUsing({ source: 'The Diamond Weapon', id: '5FA2', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ source: 'Diamant-Waffe', id: '5FA2', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ source: 'Arme Diamant', id: '5FA2', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ source: 'ダイヤウェポン', id: '5FA2', capture: false }),
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'East claw -> Spread',
+        },
+      },
+    },
+    {
+      id: 'DiamondEx Photon Burst',
```

Comment:
Can you leave a TODO (or change this if you want) to do a "collect who has the flare markers" -> tell people with flare that they have flare and tell other people to get away from flare?

Reply:
Flares go on top two enmity targets, usually tanks unless things are really going south. They'll kill anyone else they target (unless that person is really shielded), since they do about 170k without mitigation. Since tanks should usually have the flares, I didn't really see a need to do a collect setup for it.

Reply:
If you don't think this should be a trigger, I am happy to write a followup to do it.  I think it is very easy for tanks to die or to forget to provoke when they get rez'd, and some random dps gets a flare.  And that dps with a flare, probably needs to jump or get out of the group or to do something.

Reply:
I'm fine with a follow up for this one. I've added a TODO above the trigger.